Library/Homebrew/download_strategy.rb

```diff
@@ -2,6 +2,9 @@
 require "rexml/document"
 require "time"
 
+# Superclass for all the download strategies, has 2 subclasses
+# VCSDownloadStrategy and AbstractFileDownloadStrategy
+# They are further classified later
```

Comment:
How about just `Superclass for all other download strategies`. Don't want to name specific classes as the documentation will become outdated if the code changes.